.gitlab-ci.yml

```diff
@@ -229,6 +229,26 @@ build:bleeding-edge:
     EXTRA_CONF: "-native-compiler yes -coqide opt"
     EXTRA_PACKAGES: "$COQIDE_PACKAGES"
 
+build:windows:
+  stage: test
+  artifacts:
+    name: "$CI_JOB_NAME"
+    paths:
+      - dev\nsis\*.exe
+      - coq-opensource-archive-windows-*.zip
+    expire_in: 1 week
+  dependencies: []
+  tags:
+    - windows
+  before_script:
+    - SET ARCH=64
+  script:
+    - powershell -Command "(New-Object Net.WebClient).DownloadFile('http://www.cygwin.com/setup-x86_64.exe', 'setup-x86_64.exe')"
+    - call dev/ci/gitlab.bat
+  only:
+    variables:
+      - $WINDOWS == "enabled"
```

Comment:
It seems the job wasn't created, did you set the variable after pushing or is there a problem?
(also the build job failed and you need to rebase)

Reply:
Yeah I set it after. I should probably relaunch it.